templates.i.md

```diff
@@ -30,6 +40,13 @@ CxSMILES `Cl*.c1ccccc1-c1ccccc1 |m:1:3.4.5.6.7.8.9|`:
 <img src="./images/generated/monochlorobiphenyl.svg" width="400" alt="SVG depiction of biphenyl with a single chloride atom at an unknown position" />
 <out>TemplateMonobiphenyl</out>
 
+### Flavonoid 
+
+Or for a flavonoid with known number of hydroxy groups on each rings, but with position uncertainty:
+`O*.O*.C1=C(C(=C2C(=C1)OC(=CC2=O)C3=CC(=C(C=C3)))).O* |m:3:4.5,m:1:8.9,m:22:18.19|`
+<img src="./images/generated/flavonoid.svg" width="400" alt="SVG depiction of flavonoid with known hydroxy groups on each cycles but position uncertainty" />
+<out>TemplateFlavonoid</out>
```

Comment:
Plz make a copy of https://github.com/egonw/cdk-cxsmiles/blob/master/code/TemplateLipidDoubleBond.groovyin and have the `<out>` value match the filename (minus `.groovyin`) and update in that file lines 16 (with the CxSMILES) and 21 (with the SVG filename), and have the latter match what's in the `<img src>` in this file.